src/components/settings/ContractVersion/index.tsx

```diff
@@ -42,6 +45,27 @@ export const ContractVersion = ({ isGranted }: { isGranted: boolean }) => {
           Unsupported contract
         </Typography>
       )}
+      <>
+        <br></br>
+        <br></br>
+        {isUpdated! ? (
+          <Typography variant="body1" fontWeight={400}>
+            <Typography variant="body1" fontWeight={400}>
+              ✅ Latest version
+            </Typography>
+          </Typography>
+        ) : (
+          <Tooltip
+            title="Update now to take advantage of new features and the highest security standards available.
+You will need to confirm this update just like any other transaction."
+            placement="right-end"
+          >
+            <Link sx={{ fontSize: '14px' }} href="">
```

Comment:
> `sx={{ fontSize: '14px' }}`

@iamacook do you know if we can use `variant="body2"` here or something similar?

Reply:
Yes, `Link` [accepts `variant`](https://mui.com/material-ui/api/link/#props).